### PR TITLE
More data viz polishings!

### PR DESCRIPTION
**Pull request by cansavvy**
## Summary 

This is stacked on to #542 Because I wanted to chunk up the changes 

This addresses the following from https://github.com/jhudsl/intro_to_r/issues/530

> * [x]  adding a slide before tips to show that we are now talking about tips and common issues might be good
> * [x]  maybe we can remove the linetype from the plots until we get to introducing it

I just added a line that explains it. See how you feel about that strategy. 

> * [x]  mention ThemePark maybe in the lecture
> * [x]  revisit slide before axis limits with xlim() and ylim()- do we need that there now?
> * [x]   add a tip that you need to run the name of your plot to see it - someone got confused about this
> * [x]  add that you can use numbers for linetype too
> Lab
> * [x]  add `themepark` to show fun themes :)  https://matthewbjane.com/ThemePark/ - need to tell people how to install
> * [x]  add tip about using `theme(legend.position = "none")` for the last bonus in the lab - not sure what my last issue was saying about this- maybe I had already fixed it but I think a bigger tip might be helpful. or actually maybe it is best to leave it out but provide tips on how to find the documentation to figure it out
> [x] get rid of `mapping= aes`.. throughout? We weren't using it consistently

I wasn't really sure what these ones mean from that issue so they are not yet addressed: 
> * [x]  for final question (not bonus) see what facet looks like with and without geom_point() - should lose a value for one of the facets and it will be empty
> * [ ]  explain what groups are doing some more in the lecture - splitting up a line where multiple values are being plotted etc.

### CODE REVIEW

**Comment by cansavvy** [comment]
This https://github.com/jhudsl/intro_to_r/pull/546/files needs to be tested and merged first. Then the docker image needs to be updated.

File: modules/Data_Visualization/Data_Visualization.html
Comment (avahoffman) [inline modules/Data_Visualization/Data_Visualization.html:1]: html files shouldn't be committed (similar to OTTR)

Comment (cansavvy) [reply modules/Data_Visualization/Data_Visualization.html:1]: Okay cool. Its this stuff I wasn't sure about the proper contributing etiquette on this repo. 👍

Comment (avahoffman) [reply modules/Data_Visualization/Data_Visualization.html:1]: Of course! Should be more clear with updates to the contributing guide in #551 ❤️

**Comment by avahoffman** [comment]
Apologies for intruding - wanted to help get this caught up, especially given you've been bogged down. 

Added question to address the second to last to do above. The last one is something that is already done, but could perhaps be improved. I think we are okay skipping that. 

This is ready to go!

File: modules/Data_Visualization/Data_Visualization.html
Comment (cansavvy) [reply modules/Data_Visualization/Data_Visualization.html:1]: OH Noice.

**Comment by cansavvy** [comment]
> Apologies for intruding - wanted to help get this caught up, especially given you've been bogged down.

Oh no need for apologies. My feeling is one of relief -- my toes feel completely step free. 

Happy to just continue to tee things up while you do the running into home. No problem by me if this is what you find helpful!

**Pull request merged**